docs/guides/getting-started-orchestrate-human-tasks.md

```diff
@@ -6,94 +6,208 @@ description: "Efficiently allocate work through user tasks."
+### Step 5: Complete a user task
 
-:::note
-Here, after implementing your gateway, is when [expressions](/components/concepts/expressions.md) will become useful in accessing [variables](/components/concepts/variables.md) and calculating their value to move forward in a process.
+When the process instance arrives at the user task, a new job is created. The process instance stops at this point and waits until the job is completed. Applications like [Tasklist](/components/tasklist/introduction-to-tasklist.md) can be used by humans to complete these tasks. In this last step, you will open Tasklist to run the user task you created.
+
+:::tip
+While it may originally seem like the goal of automating a process is to remove humans entirely, efficiently allocating work through user tasks can be even more beneficial. Within this example, we've included a form to demonstrate the completion of a user task.
+
+Using the Zeebe or Tasklist API, many other ways to complete a user task are possible, such as redirecting to another application to complete the task, or even listening to IoT devices to capture human interaction with the real world via job workers.
 :::
 
+1. Click the **navigation menu icon** next to the Camunda logo in the top bar to open the global navigation.
+2. Click **Tasklist** to open the Tasklist application.
+   <img src={ModelerNavImg} style={{width: 200}} alt="Navigation to other applications" />
+
+3. On the left, you will notice a list of **tasks**. There should be one open task `Decide what's for dinner`. Click this task to open it in the detail view.
+4. In the detail view, the form you created in [step 2](step-2-design-a-form) appears. It is read only since this task is currently unassigned. You have to claim the task to work on it. Next to **Assignee**, click **Assign to me** to claim the task.
+5. Select one of the radio options.
+6. Click **Complete Task** to submit the form.
+   ![complete a human task in Tasklist](./img/user-task-tasklist.png)
+7. To verify your task completion, you can filter by **Completed** tasks in the left task list panel.
+
+You can now navigate back to Operate and notice the process instance has continued as the token has moved forward to the selected option.
+
+The token moves through the exclusive gateway (also called the XOR gateway), and is used to model the decision in the process. When the execution arrives at this gateway, all outgoing sequence flows are evaluated in the order in which they have been defined. The sequence flow which condition evaluates to ‘true’ is selected for continuing the process.
```

Comment:
@christinaausley I tried to reduce links to other pages in the main content of the guide since my goal is to increase completion rate of the guide. Our metrics show that only a few readers are completing the guide by creating this process in Modeler after they opened the guide, and many do not come back to the guide when navigating to other pages, dropping off completely. Hence, I added most links to the bottom of the page in the "read more" section

Reply:
Good with testing this out 👍